web/views/pages/index.ejs

```diff
@@ -224,60 +224,57 @@ co2Sub = function(str) {
-                                <p>
-                                    <!-- Facebook share -->
-                                    <div class="fb-share-button"
-                                        data-href="https://www.electricitymap.org/"
-                                        data-layout="button_count">
-                                    </div>
-                                    <!-- Twitter share -->
-                                    <a class="twitter-share-button"
-                                        data-url="https://www.electricitymap.org"
-                                        data-via="electricitymap"
-                                        data-lang="<%= locale %>">
+                            <p>
+                                <!-- Facebook share -->
+                                <div class="fb-share-button"
+                                    data-href="https://www.electricitymap.org/"
+                                    data-layout="button_count">
+                                </div>
+                                <!-- Twitter share -->
+                                <a class="twitter-share-button"
+                                    data-url="https://www.electricitymap.org"
+                                    data-via="electricitymap"
+                                    data-lang="<%= locale %>">
+                                </a>
+                                <!-- Slack -->
+                                <span class="slack-button">
+                                    <a href="https://slack.tmrow.co" target="_blank" class="slack-btn">
+                                        <span class="slack-ico"></span>
+                                        <span class="slack-text">Slack</span>
                                     </a>
-                                    <!-- Slack -->
-                                    <span class="slack-button">
-                                        <a href="https://slack.tmrow.co" target="_blank" class="slack-btn">
-                                            <span class="slack-ico"></span>
-                                            <span class="slack-text">Slack</span>
-                                        </a>
-                                    </span>
-                                </p>
+                                </span>
+                            </p>
                         </div>
 
                         <div class="mobile-faq-header">
-                                <%- __('misc.faq') %>
-                        </div>
-                        <div class="mobile-faq">
+                            <%- __('misc.faq') %>
                         </div>
+                        <div class="mobile-faq"></div>
                     </div>
                     <div class="left-panel-zone-details">
 
-                        <div class="left-panel-zone-details-toolbar">
```

Comment:
moved the toolbar (country name and flag) into the header (which is always visible).
Not sure why this wasn't done before.

Reply:
I think I remember this being a bit of a compromise to not have too much vertical space being taken up by fixed content, reducing the amount of info visible on smaller screens.